src/components/NcAppNavigationCaption/NcAppNavigationCaption.vue

```diff
@@ -242,7 +242,6 @@ export default {
 		// padding to align the name with the icon of app navigation items
 		padding: 0 calc(var(--default-grid-baseline, 4px) * 2) 0 calc(var(--default-grid-baseline, 4px) * 2);
 		margin-top: 0px;
-		margin-bottom: 12px;
```

Comment:
Does not work on Nextcloud 30 as we have default styles for headings, there should be no margin at all then use

```css
    margin-bottom: 0px;
```

But I would rather go with:
```css
    margin-bottom: var(--default-grid-baseline);
```

Because otherwise the text of the heading end directly on the outline of the next list entry.

Reply:
Done